docs/contributing.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+(For now, just some notes to ick authors)
+
+## Logging
+
+We use vmodule, which provides extra logging levels.  Use `VLOG_1` to mean "info,
+but more" and `VLOG_2` to mean "info, but even more."
+
+DEBUG messages should be only of interest to ick authors, whereas VLOG_1 and
```

Comment:
We don't provide any specific recommendation for the count you can use from one level to the next, but do treat INFO in the glog style of being an opt-in for debugging, not something users would _normally_ see.